lte/gateway/c/oai/lib/pcef/pcef_handlers.cpp

```diff
@@ -174,12 +174,15 @@ bool pcef_end_session(char* imsi, char* apn) {
 
 void pcef_send_policy2bearer_binding(
     const char* imsi, uint8_t default_bearer_id, char* policy_rule_name,
-    uint8_t eps_bearer_id) {
+    uint8_t eps_bearer_id, uint32_t eps_bearer_agw_teid,
```

Comment:
minor: can these be const parameters?

Reply:
oh I see now that `policy_rule_name` is not part of this PR :D

Reply:
yep, good catch...

Reply:
I changed all the inputs as const type.